### ISSUE TITLE
Model: Best model validation

### ISSUE THREAD

**Issue by kaqreal** [kaqreal/ds-Capstone_Fish_Project#31]

**Comment by kaqreal** [comment]
Best model: Model maker (CNN model with transfer learning = using webscrapping and species) = 43% accuracy
(https://www.tensorflow.org/lite/models/modify/model_maker/image_classification)

Best model: Model maker (CNN model with transfer learning = with hypertuning and families only) = 59% accuracy
(https://www.tensorflow.org/lite/models/modify/model_maker/image_classification)

Other models (already tried):
- Vision Transformer (https://keras.io/examples/vision/image_classification_with_vision_transformer/)
- Classify ImageNet classes with (https://keras.io/api/applications/#usage-examples-for-image-classification-models):
1. ResNet50
2. VGG16
3. VGG19
4. Fine-tune InceptionV3

Base model (to be completed with a link)
